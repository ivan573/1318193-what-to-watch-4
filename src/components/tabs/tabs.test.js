import React from "react";
import renderer from "react-test-renderer";
import Tabs, {TabOption} from "./tabs.jsx";

describe(`Tabs match snapshots`, () => {
  it(`Overview tab matches snapshot`, () => {
    const tree = renderer
      .create(<Tabs
        activeTab={TabOption.OVERVIEW}
      />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Details tab matches snapshot`, () => {
    const tree = renderer
      .create(<Tabs
        activeTab={TabOption.DETAILS}
      />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Reviews tab matches snapshot`, () => {
    const tree = renderer
      .create(<Tabs
        activeTab={TabOption.REVIEWS}
      />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
