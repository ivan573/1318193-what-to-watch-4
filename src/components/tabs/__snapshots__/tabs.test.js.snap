// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs match snapshots Details tab matches snapshot 1`] = `[Function]`;

exports[`Tabs match snapshots Overview tab matches snapshot 1`] = `[Function]`;

exports[`Tabs match snapshots Reviews tab matches snapshot 1`] = `[Function]`;
